File: programmers/cpp/기초100/2.5_문자열겹쳐쓰기/2-5.cpp
#include <string>
#include <vector>

using namespace std;

string solution(string my_string, string overwrite_string, int s) {
    string answer = "";
    
    // 코드입력
    answer = my_string.replace(s,overwrite_string.size(),overwrite_string);

    return answer;
}

File: programmers/java/기초124/2.5_문자열겹쳐쓰기/2-5.java
class Solution {
    public String solution(String my_string, String overwrite_string, int s) {
        // String answer = "";
        // return answer;
        //////////////////////////////////////
        String front = my_string.substring(0, s); // 인덱스 s부터 overwrite_string의 길이만큼
        String back = my_string.substring(s + overwrite_string.length()); // overwrite_string으로 바꾼 문자열
        return front + overwrite_string + back;
        //////////////////////////////////////
    }
}